src/UbiConstants.h

```diff
@@ -30,14 +30,13 @@ Inc
 #include "stdint.h"
 
 const char UBIDOTS_INDUSTRIAL_IP[] = "169.55.61.243";
-const char *const USER_AGENT = "UbidotsSIM900/4.0.1";
-const int UBIDOTS_HTTP_PORT = 80;
-const int UBIDOTS_HTTPS_PORT = 443;
+const char *const USER_AGENT = "SIM900";
```

Comment:
user agent must follow the specification already written here https://ubidots.quip.com/TzVbA1mGsbYc/User-Agent-Creation-Standardization